packages/react/src/SubdomainNavBar/SubdomainNavBar.tsx

```diff
@@ -339,9 +339,9 @@ const _SearchInternal = (
     setLiveRegion(true)
 
     setTimeout(() => {
-      setLiveRegion(false)
+      if (active) setLiveRegion(false)
```

Comment:
As this is a functional change, should we have another test for this now? Coverage will presumably have dropped on this branching logic otherwise.